README.md

```diff
@@ -145,9 +158,9 @@ When the `machete.slideOut.deleteLocalBranch` git configuration key is set to `t
 The easiest way to set this key is using `git config --add machete.slideOut.deleteLocalBranch true` command inside given repository.
 If you want to set this key globally (to by used for all repositories that don't have this key set explicitly), add `--global` option to the previous command.
 
-![](docs/slide_out1.gif)
+![](docs/slide_out_leaf_node.gif)
 
-![](docs/slide_out2.gif)
+![](docs/slide_out_non_leaf_node.gif)
 
 ### Slide in branch
```

Comment:
Hmm is it possible in GH markdown to make section collapsible (as on e.g. Wikipedia?)

Reply:
Maybe this https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab, TBD how to structure it so that stuff is both easily accessible & visible to newcomers, and the entire README isn't too long to scroll

Reply:
Just as I'm now doing in https://github.com/VirtusLab/git-machete/tree/doc/readme-first-5-mins